server.js

```diff
@@ -1,265 +1,251 @@
-"use strict"; 
-require('dotenv').load();
+'use strict'
+require('dotenv').load()
 
-var WebSocketServer = require('websocket').server;
-var http = require('http');
-var HttpDispatcher = require('httpdispatcher');
-var dispatcher = new HttpDispatcher();
-var url = require("url");
+var WebSocketServer = require('websocket').server
+var http = require('http')
+var HttpDispatcher = require('httpdispatcher')
+var dispatcher = new HttpDispatcher()
+var url = require('url')
 
-const fs = require('fs');
+const fs = require('fs')
 
-const Speech = require('@google-cloud/speech');
-
-const speech = new Speech.SpeechClient();
+const Speech = require('@google-cloud/speech')
 
+const speech = new Speech.SpeechClient()
 
 var clients = []
 
-//Create a server
-var server = http.createServer(handleRequest);
-var wsserver = http.createServer(handleRequest);
-
-process.env.APP_ID
+// Create a server
+var server = http.createServer(handleRequest)
+var wsserver = http.createServer(handleRequest)
 
-const Nexmo = require('nexmo');
+const Nexmo = require('nexmo')
 const nexmo = new Nexmo({
-  apiKey: "dummy",
-  apiSecret: "dummy",
+  apiKey: 'dummy',
+  apiSecret: 'dummy',
```

Comment:
load this in via `process.env.NEXMO_API_SECRET`

Reply:
Seems that having App ID and private key makes it so that these are not needed.

Reply:
Would need to see how the library behaves when only getting private key and App ID. But, yeah, that'd make sense.